MaterialDesignThemes.Wpf/DialogHost.cs

```diff
@@ -181,6 +181,22 @@ public static async Task<object> Show(object content, object dialogIdentifier, D
             return await targets[0].ShowInternal(content, openedEventHandler, closingEventHandler);
         }
 
+        /// <summary>
+        ///  Close a modal dialog.
+        /// </summary>
+        /// <param name="dialogIdentifier"> of the instance where the dialog should be shown. Typically this will match an identifer set in XAML. </param>
+        public static void CloseDailog(object dialogIdentifier)
+        {
+            if (dialogIdentifier == null) throw new ArgumentNullException(nameof(dialogIdentifier));
+
+            if (LoadedInstances.Count == 0)
+                throw new InvalidOperationException("No loaded DialogHost instances.");
+
+            var targets = LoadedInstances.FirstOrDefault(dh => dialogIdentifier == null || Equals(dh.Identifier, dialogIdentifier));
+            if (targets != null)
+                CloseDialogCommand.Execute(false, null);
```

Comment:
You can call `Close()` instead of `CloseDialogCommand.Execute()`. Note that need to check whether `CurrentSession` is not null.